modules/ROOT/partials/misc/mobile-platform-compatibility.adoc

```diff
@@ -1,12 +1,39 @@
-== Supported Mobile Platforms
+== Supported Mobile Platforms (1)
 
-{productname} mobile is designed to run on iOS Safari and Android Chrome. {productname} mobile is tested on the following platforms:
+{productname} mobile is tested against, and is designed to run on, the default browser running on a currently vendor-supported version of both Android and iOS.
 
-[cols="^,^,^",options="header"]
+[cols="1,1",options="header"]
 |===
-|OS |Browser |Device
-|Android 11 |Chrome |Mobile phone
-|Android 12 |Chrome |Mobile phone
-|iOS/iPadOS 14 |Safari |iPhone/iPad
-|iOS/iPadOS 15 |Safari |iPhone/iPad
+|Operating System |Default browser
+
+|Android
+|Chrome
+
+|iOS
+|Safari
 |===
+
+Google supports the current and two previous versions of Android.
+
+Apple supports the current and previous versions of iOS.
```

Comment:
How many previous versions of iOS? Is it known/consistent?

Reply:
The out-of-date version of this section (and previous versions of this section) always listed the then-current and then-penultimate versions of both iOS and Android.

Version (2) of this section preserves this but re-writes it such that the section does not need constant updating.

Version (1), by contrast, has us supporting TinyMCE on all vendor-supported versions of Android and iOS.

The vendor-supported versions are the current and penultimate versions of iOS (the two versions Apple supports) and the current, penultimate and ante-penultimate versions of Android (the three versions Google supports).

Version (2) is the status quo re-phrased to no longer require constant updating.

Version (1) is a change that would require confirmation of this actually being the case. But would also bring how we support TinyMCE on Android in line with how Google supports Android on mobile devices.

Reply:
Is removing the "Google supports..." and "Apple supports..." lines, but preserving the phrase "vendor-supported" a good compromise? The assumption is that people will know or research which versions are vendor-supported themselves.

Reply:
>Is removing the “Google supports…” and “Apple supports…” lines, but preserving the phrase “vendor-supported” a good compromise? The assumption is that people will know or research which versions are vendor-supported themselves.

Simpler is a good argument for removing the lines, but it still requires confirmation that we do, in fact, support Android through all three versions Google supports, rather than the two (current and penultimate) we always, previously, said we supported.

FWIW, the reason for bringing up the notion of matching our support with what the venders supported is that vendor-support is commonly a measure used by companies for what they, themselves, support.

If a company deploys a mobile app which incorporates TinyMCE and they have Android devices in staff hands that are running what is, at the time of deployment, the ante-penultimate version of Android, our not supporting TinyMCE on that device while Google still does support said device might not sit well with said company.

Reply:
With those two lines removed I still believe it is an improvement over version 2 though. Regardless, I support version 1 even if it means we have to ensure that the two lines are up to date - I think if it ever changes it will be big enough news that we don't have to explicitly search for and keep track of it.